distributed/tests/test_zstd.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+
+@pytest.mark.parametrize(
+    "cfg,values",
+    [
+        # 'cffi' is NOT support by "pip install zstandard"
+        # but AVAILABLE by manual install zstandard with "python setup.py install"
+        ("distributed.comm.zstd.import_policy", ["default", "cffi_fallback", "cext"]),
+        ("distributed.comm.zstd.level", range(1, 23)),
+        ("distributed.comm.zstd.write_checksum", [False, True]),
+        ("distributed.comm.zstd.write_content_size", [True]),
+        ("distributed.comm.zstd.write_dict_id", [False, True]),
+        ("distributed.comm.zstd.threads", range(-1, 100, 1)),
+    ],
+)
+def test_zstd(cfg, values):
+    zstandard = pytest.importorskip("zstandard")
+    import zstandard as zstd, six, cffi, dask, distributed.protocol.compression as compr
+
+    original_data = b"".join(
+        [
+            b"\xe6\x88\x91\xe6\x98\xaf\xe4\xb8\xad\xe5\x9b\xbd\xe4\xba\xba\xe6\xb0",
+            b"\x91\xe7\x9a\x84\xe5\x84\xbf\xe5\xad\x90\xef\xbc\x8c\xe6\x88\x91\xe6",
+            b"\xb7\xb1\xe6\x83\x85\xe5\x9c\xb0\xe7\x83\xad\xe7\x88\xb1\xe7\x9d\x80",
+            b"\xe6\x88\x91\xe7\x9a\x84\xe7\xa5\x96\xe5\x9b\xbd\xe5\x92\x8c\xe4\xba",
+            b"\xba\xe6\xb0\x91\xe3\x80\x82",
+        ]
+    )
```

Comment:
Why this data in particular?  Is it just random data?  If so, lets use the following:

```python
import numpy as np

original_data = np.random.random(10).data
```

Reply:
it's a sample bytes that long enough.
random data is ok for short size.

Reply:
Instead of embedding a bytestring in our tests, we should just generate one as Matt said above. As a maintainer I see this fixed bytestring as written and think "why these bytes, I wonder why they matter", whereas seeing `os.urandom(100)` it's obvious that they're just a random byte payload.